### DISCUSSION TITLE
Unexpected shutdown

### DISCUSSION THREAD

**Discussion by gdhadiya** [ant-media/Ant-Media-Server#5186] [Q&A]
some times antmedia webpanel is automatically logout from my side or some times it's shows unexpected shutdown pop-up that's why my streaming is not uploaded in server. 
any one help me?. why this happen.
![Uploading Screenshot 2023-05-26 at 3.34.20 PM.png…]()

**Comment by Mohit-3196** [comment]
Hi @gdhadiya ,
The unexpected shutdowns mostly happen due to high resource usage.
The screenshot that you tried uploading hasn't been uploaded so I can't see the image.

If you see the ant-media-server.log file there you can see some high resource usage warnings. Also, when unexpected shutdowns happen, you can raise a support ticket with the logs so that we can check it and understand the exact reason for such occurrences.

Thank you
